### PR TITLE
#161290997 Deploy base app to Heroku

### PR DESCRIPTION
**Pull request by Darthrighteous**
#### What does this PR do?
Set up app Heroku pipeline

#### Description of Task to be completed?
- have staging and production apps setup on Heroku
- have commits on develop auto-deploy to staging app
- have review app for PRs set up

#### How should this be manually tested?
- view app on https://valinor-ah-backend-staging.herokuapp.com/

#### Any background context you want to provide?
I have deleted the db connection logic in the root `index.js` file. I did this because it uses an env variable `MONGODB_URI` which doesn't currently exist in the production environment, seeing as we're going to be using postgres for the project and this was preventing the app from running.

#### What are the relevant pivotal tracker stories?
[#161290997](https://www.pivotaltracker.com/story/show/161290997)

#### Screenshots (if appropriate)
screen shot 2018-10-25 at 10 40 33 pm

#### Questions:
N/A

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -38,12 +38,12 @@ if (!isProduction) {
     app.use(errorhandler());
 }
 
-if (isProduction) {
-    mongoose.connect(process.env.MONGODB_URI);
-} else {
+// if (isProduction) {
```
Comment (andela-moe) [inline index.js]: I would rather suggest you remove lines of code that are not being used versus commenting them. Any reason why we have left this here? Do we hope to reuse it sometime in the future for this application?

Comment (Darthrighteous) [reply index.js]: @andela-moe I'll do that, I don't think we'll have need to reuse any of that code.

File: app.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      "quantity": 1
+    }
+  },
+  "addons": [],
+  "buildpacks": [
+    {
+      "url": "heroku/nodejs"
+    }
+  ]
+}
```
Comment (KvNGCzA) [inline app.json]: Add a new line here

Comment (Darthrighteous) [reply app.json]: thanks, will do

**Pull request merged**